README.md

```diff
@@ -68,7 +68,103 @@ With this framework you need to record the source and target metadata in an onbo
 
 ## Getting Started
 Refer to the [Getting Started](https://databrickslabs.github.io/dlt-meta/getting_started)
+### Databricks Labs DLT-META CLI lets you run onboard and deploy in interactive python terminal
+#### pre-requisites:
+- [Databricks CLI](https://docs.databricks.com/en/dev-tools/cli/tutorial.html)
+- Python 3.8.0 +
+#### Steps:
+- ``` git clone dlt-meta ```
+- ``` cd dlt-meta ```
+- ``` python -m venv .venv ```
+- ```source .venv/bin/activate ```
+- ``` pip install databricks-sdk ```
+- ```databricks labs dlt-meta onboard``` 
+- - Above command will prompt you to provide onboarding details. If you have cloned dlt-meta git repo then accept defaults which will launch config from demo folder.
 
+```     Provide onboarding file path (default: demo/conf/onboarding.template): 
+        Provide onboarding files local directory (default: demo/): 
+        Provide dbfs path (default: dbfs:/dlt-meta_cli_demo): 
+        Provide databricks runtime version (default: 14.2.x-scala2.12): 
+        Run onboarding with unity catalog enabled?
+        [0] False
+        [1] True
+        Enter a number between 0 and 1: 1
+        Provide unity catalog name: ravi_dlt_meta_uc
+        Provide dlt meta schema name (default: dlt_meta_dataflowspecs_203b9da04bdc49f78cdc6c379d1c9ead): 
+        Provide dlt meta bronze layer schema name (default: dltmeta_bronze_cf5956873137432294892fbb2dc34fdb): 
+        Provide dlt meta silver layer schema name (default: dltmeta_silver_5afa2184543342f98f87b30d92b8c76f): 
+        Provide dlt meta layer
+        [0] bronze
+        [1] bronze_silver
+        [2] silver
+        Enter a number between 0 and 2: 1
+        Provide bronze dataflow spec table name (default: bronze_dataflowspec): 
+        Provide silver dataflow spec table name (default: silver_dataflowspec): 
+        Overwrite dataflow spec?
+        [0] False
+        [1] True
+        Enter a number between 0 and 1: 1
+        Provide dataflow spec version (default: v1): 
+        Provide environment name (default: prod): prod
+        Provide import author name (default: ravi.gawai): 
+        Provide cloud provider name
+        [0] aws
+        [1] azure
+        [2] gcp
+        Enter a number between 0 and 2: 0
+        Do you want to update ws paths, catalog, schema details to your onboarding file?
+        [0] False
+        [1] True
+```
+- Goto your databricks workspace and located onboarding job under: Workflow->Jobs runs
```

Comment:
The `databricks labs dlt-meta onboard` command seems to exit without any confirmation as shown below:
```
Update workspace/dbfs paths, unity catalog name, bronze/silver schema names in onboarding file?
[0] False
[1] True
Enter a number between 0 and 1: 1
(.venv) ➜  dlt-meta git:(feature/dlt-meta-uc) 
```
I didn't know whether the command completed successfully or not. I had to read this README to figure out I have to go check the workflow UI to find the onboarding job. 
We should at least try to print a successful message with the expected job name. It would be great if we can also print the job URL.

Reply:
I had prints but @nfx suggested to remove since labs cli will print it as part of logs.